graphql/src/query/mod.rs

```diff
@@ -58,6 +61,16 @@ where
             Err(errors) => return QueryResult::from(errors),
         };
 
+    let mode = if let q::OperationDefinition::Query(query) = operation {
+        if query.directives.iter().any(|dir| dir.name == "verify") {
```

Comment:
We don't want it as an environment variable since that would slow down _every_ query that the node runs; I am trying to provide a simple way to check that some complicated query that somebody dreamed up leads to the same result as before as easily as possible.

Reply:
Yeah, I agree now.